diff --git a/scripts/pinned_build.sh b/scripts/pinned_build.sh
index 6e4ee3787..40cc16c15 100755
--- a/scripts/pinned_build.sh
+++ b/scripts/pinned_build.sh
@@ -133,7 +133,6 @@ echo "Building Leap ${SCRIPT_DIR}"
 try cmake -DCMAKE_TOOLCHAIN_FILE=${SCRIPT_DIR}/pinned_toolchain.cmake -DCMAKE_INSTALL_PREFIX=/usr/local -DCMAKE_BUILD_TYPE=Release -DCMAKE_PREFIX_PATH=${LLVM_DIR}/lib/cmake -DCMAKE_PREFIX_PATH=${BOOST_DIR}/bin ${SCRIPT_DIR}/..
 
 try make -j${JOBS}
-try cpack
 
 echo " .----------------.  .----------------.  .----------------.  .----------------. ";
 echo "| .--------------. || .--------------. || .--------------. || .--------------. |";
